documentation/tutorials/getting-started-with-ash-authentication-phoenix.md

```diff
@@ -1,87 +1,365 @@
 # Getting Started Ash Authentication Phoenix
 
-This guide assumes that you already have an Phoenix application set up with Ash.
-If you don't then check out the [Phoenix topic on Ash
-HQ](https://ash-hq.org/docs/guides/ash/latest/topics/phoenix).
+In this 10 minute tutorial we create a new empty `Example` Phoenix application which provides the functionality to register users with their email address and a password. Registered users can sign in and out.
 
+We assumes that you have [Elixir](https://elixir-lang.org) version 1.14.x (check with `elixir -v`) and Phoenix 1.7 (check with `mix phx.new --version`) installed. We also assume that you have a [PostgreSQL](https://www.postgresql.org) database running which we use to persist the user data.
 
-## Before continuing
+## Green Field
 
-If you haven't already, read and follow [Getting Started with Ash Authentication](https://ash-hq.org/docs/guides/ash_authentication/latest/tutorials/getting-started-with-authentication). You will use the things you create there in this tutorial.
+We start with a new Phoenix application and use the `--no-ecto` flag to skip the [Ecto](https://hexdocs.pm/ecto/Ecto.html) setup.
 
-## Add to your application's dependencies
+```bash
+$ mix phx.new example --no-ecto
+$ cd example
+```
+
+## Application Dependencies
+
+We need to add the following dependencies:
+
+** `mix.exs` **
 
 ```elixir
-# mix.exs
+defmodule Example.MixProject do
+  use Mix.Project
+  # ...
 
-defp deps()
-[
+    defp deps do
+    [
+      # ...
+      # add these lines -->
+      {:ash, "~> 2.5.11"},
+      {:ash_authentication, "~> 3.7.3"},
+      {:ash_authentication_phoenix, "~> 1.4.5"},
+      {:ash_postgres, "~> 1.3.2"},
+      {:elixir_sense, github: "elixir-lsp/elixir_sense", only: [:dev, :test]} 
```

Comment:
This part should become unnecessary as soon as a new elixir-ls release happens. Been waiting a while on that, though.